scout/commands/update/case.py

```diff
@@ -197,6 +197,7 @@ def case(
 
     if case_changed:
         institute_obj = store.institute(case_obj["owner"])
+        case_obj["status"] = "inactive"
```

Comment:
You could also trigger it from each single function in the cli that loads variants for each type. But like this is so much cleaner! But I'm open for suggestions